sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2357,6 +2398,74 @@ pi_result cuda_piextKernelSetArgSampler(pi_kernel kernel, pi_uint32 arg_index,
   return retErr;
 }
 
+pi_result cuda_piKernelGetGroupInfo(pi_kernel kernel, pi_device device,
+                                    pi_kernel_group_info param_name,
+                                    size_t param_value_size, void *param_value,
+                                    size_t *param_value_size_ret) {
+
+  // here we want to query about a kernel's cuda blocks!
```

Comment:
```suggestion
  // Here we want to query about a kernel's cuda blocks!
```

Reply:
I've changed this with the newest commit.